packages/models/src/migrations/20230406150645-create-activity.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('activity', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      valid: {
+        type: Sequelize.BOOLEAN,
+        defaultValue: true,
+      },
+      points: {
+        type: Sequelize.INTEGER,
+      },
+      currency: {
+        type: Sequelize.BLOB,
+      },
+      price: {
+        type: Sequelize.DECIMAL,
+      },
+      priceUSD: {
+        type: Sequelize.FLOAT,
+        field: 'price_usd',
+      },
+      type: {
+        type: Sequelize.STRING,
+      },
+      contractAddress: {
+        type: Sequelize.BLOB,
+        field: 'contract_address',
+      },
+      walletAddress: {
+        type: Sequelize.BLOB,
+        field: 'wallet_address',
+      },
+      activityHash: {
```

Comment:
How about having a unique constraint on this column?

Reply:
like this?  

https://github.com/OriginProtocol/storypoints/blob/6584e5f5f6beddbe13b2f8f455126f85c2a0c980/packages/models/src/migrations/20230406150645-create-activity.js#L75-L79